jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -191,7 +191,7 @@ private Image pullBaseImage(
     EventHandlers eventHandlers = buildContext.getEventHandlers();
 
     ManifestAndDigest<?> manifestAndDigest =
-        registryClient.pullManifest(buildContext.getBaseImageConfiguration().getImageTag());
+        registryClient.pullManifest(buildContext.getBaseImageConfiguration().getImageQualifier());
```

Comment:
I think this answers some of my earlier questions. When given both a tag and a digest, the end result is to ignore a tag, right?

If so, since there's no point in specifying a tag in a Jib config, maybe we should print a warning. This doesn't have to be addressed in this PR.

Reply:
I will file an issue around these questions. Ultimately, we should guard accidental misuse and confusion.